File: xsdelements/elementswrapper/ConcreteElement.java
package proholz.xsdparser;



/**
 * ConcreteElement is a wrapper class for an {@link XsdAbstractElement} object which doesn't have a ref attribute.
 */
public class ConcreteElement extends ReferenceBase {

    ConcreteElement(XsdAbstractElement element){
        super(element);
    }

}

File: xsdelements/elementswrapper/ConcreteElement.pas
namespace proholz.xsdparser;

interface

type
  ConcreteElement = public class(ReferenceBase)
  assembly
	constructor(element: XsdAbstractElement);
  end;

implementation

constructor ConcreteElement(element: XsdAbstractElement);
begin
  inherited constructor(element);
end;

end.